### ISSUE TITLE
Connect FIlter Menu Community Districts dropdown to API

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#11]
## Acceptance Criteria
- [ ] Populate options in Community District dropdown in FilterMenu component ([Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=253-4689&t=i9OdRPOzwaKcOnqa-0)) using data from `findCommunityDistrictsByBoroughId` endpoint. 
- [ ] Community districts are filtered by borough based on selection in Borough dropdown.

## Related work
Blocked by
- https://github.com/NYCPlanning/ae-cp-map/issues/9

**Comment by TylerMatteo** [comment]
Closing in favor of #28
